index.html

```diff
@@ -0,0 +1,156 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
```

Comment:
Is it possible to use `./style.css` instead? :thinking:

Reply:
I didn't get it.

Reply:
Never mind, it doesn't matter here.

Reply:
Ok.